### PR TITLE
17345 Add trace to debug websocket quiesce

### LINKED ISSUE #17345: Websockets quiesce
Occasionally the websockets basic, trace, and less often cdi12 test buckets encounter a server quiesce issue after successful test execution.

### PR DESCRIPTION
**Pull request by loriadi**
fixes #17345

### CODE REVIEW

**Comment by loriadi** [comment]
fixes #17345

**Comment by loriadi** [comment]
#build

**Comment by loriadi** [comment]
#build #spawn.fullfat.buckets=io.openliberty.wsoc.internal_fat

**Comment by loriadi** [comment]
#build #spawn.fullfat.buckets=io.openliberty.wsoc.internal_fat

**Comment by loriadi** [comment]
#build #spawn.fullfat.buckets=io.openliberty.wsoc.internal_fat

**Comment by loriadi** [comment]
#build #spawn.fullfat.buckets=io.openliberty.wsoc.internal_fat

**Comment by loriadi** [comment]
#build #spawn.fullfat.buckets=io.openliberty.wsoc.internal_fat

**Comment by loriadi** [comment]
I've had no luck recreating this locally on windows or mac, and also have not recreated using the build on this PR.

I've increased the number of trace logs to keep, so that I can analyze the full set of connections when the server fails to quiesce.

File: dev/com.ibm.ws.channelfw/src/com/ibm/ws/channelfw/internal/CHFWEventHandler.java
Comment (pnicolucci) [inline dev/com.ibm.ws.channelfw/src/com/ibm/ws/channelfw/internal/CHFWEventHandler.java]: Do you want to log the actual chain name here? Something like `chain.getName()`.

We already look to log the name on entry to this method, then use the name to get all the chains and then loop through them to check if those chains are running and if so perform some actions on them.

**Review by pnicolucci — COMMENTED** [review]
Overall looks good. I have made one comment that I think needs investigation before approval.

File: dev/com.ibm.ws.channelfw/src/com/ibm/ws/channelfw/internal/CHFWEventHandler.java
Comment (mrsaldana) [reply dev/com.ibm.ws.channelfw/src/com/ibm/ws/channelfw/internal/CHFWEventHandler.java]: I agree, line 119 in this PR will log the channel name, perhaps the new log should log the current iteration ChainData's object name .

Comment (loriadi) [reply dev/com.ibm.ws.channelfw/src/com/ibm/ws/channelfw/internal/CHFWEventHandler.java]: I'll remove it since isChainRunning()
 logs the name.

**Review by loriadi — COMMENTED** [review]
I've removed that extraneous line of trace

**Comment by loriadi** [comment]
#build #spawn.fullfat.buckets=io.openliberty.wsoc.internal_fat

**Comment by loriadi** [comment]
#libby-bot

**Review by pnicolucci — APPROVED** [review]

**Pull request merged**
